packages/wdio-local-runner/src/index.ts

```diff
@@ -3,14 +3,25 @@ import { WritableStreamBuffer } from 'stream-buffers'
 
 import WorkerInstance from './worker'
 import { SHUTDOWN_TIMEOUT, BUFFER_OPTIONS } from './constants'
+import type { WorkerRunPayload } from './types'
 
 const log = logger('@wdio/local-runner')
 
+interface RunArgs extends WorkerRunPayload {
+    command: string
+    args: any
+}
+
 export default class LocalRunner {
-    constructor (configFile, config) {
-        this.configFile = configFile
+    config: WebdriverIO.Config
+    workerPool: Record<string, WorkerInstance> = {}
+
+    stdout: WritableStreamBuffer
```

Comment:
Also check if you can remove the type 👍 
```suggestion
    stdout: WritableStreamBuffer = new WritableStreamBuffer(BUFFER_OPTIONS)
```